pkgs/applications/editors/emacs/elisp-packages/manual-packages/tsc/Cargo.lock

```diff
@@ -0,0 +1,285 @@
+# This file is automatically @generated by Cargo.
```

Comment:
We shouldn't check lock files into nixpkgs: https://github.com/NixOS/nixpkgs/issues/327064.

Reply:
When there are git dependencies in `Cargo.lock`, we have to do this instead of using `cargoHash`.  https://github.com/NixOS/nixpkgs/blob/abcef4da4ebb72240bddc370a27263627e64877f/pkgs/build-support/rust/fetch-cargo-tarball/default.nix#L79-L88